solution/backend/file_manager/models.py

```diff
@@ -30,12 +30,12 @@ def __str__(self):
 
 
 class UploadedFile(models.Model):
-    document_id = models.CharField(max_length=512, null=True, blank=True)
+    document_name = models.CharField(max_length=512, null=True, blank=True)
```

Comment:
this is part of another ticket but to make it simpler and faster for others just changed here